tests/test_hazard_arbitrary_points.py

```diff
@@ -5,17 +5,14 @@
 from nzshm_common.location import CodedLocation
 from nzshm_common.location.location import LOCATIONS_BY_ID
 
-import toshi_hazard_store.query.hazard_query
-
-import nshm_hazard_graphql_api.schema.toshi_hazard.hazard_curves
+import toshi_hazard_store.query
 
 
 class TestHazardCurves:
     def test_get_hazard_curves_with_key_locations(self, mock_query_response, monkeypatch, graphql_client):
```

Comment:
what's the difference between this test and `test_get_hazard_curves_with_key_locations_lowres`?